examples/meta/src/evaluation/cross_validation_multiple_kernel_learning_weights_storage.sg

```diff
@@ -11,25 +11,25 @@ BinaryLabels labels(f_labels)
 #![create_features]
 
 #![create_kernels]
-CombinedKernel kernel()
-GaussianKernel k_1(10, 0.1)
-GaussianKernel k_2(10, 1)
-GaussianKernel k_3(10, 2)
-kernel.append_kernel(k_1)
-kernel.append_kernel(k_2)
-kernel.append_kernel(k_3)
+Kernel combined_kernel = kernel("CombinedKernel")
+Kernel k_1 = kernel("GaussianKernel", log_width=0.1)
+Kernel k_2 = kernel("GaussianKernel", log_width=1.0)
+Kernel k_3 = kernel("GaussianKernel", log_width=2.0)
+combined_kernel.add("kernel_array", k_1)
+combined_kernel.add("kernel_array", k_2)
+combined_kernel.add("kernel_array", k_3)
 #![create_kernels]
 
 #![create_classifier]
-LibSVM libsvm()
-MKLClassification svm(libsvm)
-svm.set_interleaved_optimization_enabled(False)
-svm.set_kernel(kernel)
+Machine libsvm = machine("LibSVM")
+Machine svm = machine("MKLClassification", kernel=combined_kernel)
+svm.put("interleaved_optimization", False)
```

Comment:
as said many many times before, pls put this into the kwargs
`Machine svm = machine("MKLClassification", kernel=combined_kernel, interleaved_optimization=False, svm=libsvm)